modules/net-vlan-attachment/main.tf

```diff
@@ -48,7 +48,7 @@ resource "google_compute_interconnect_attachment" "default" {
   vlan_tag8021q            = var.vlan_tag
   admin_enabled            = var.admin_enabled
   encryption               = local.ipsec_enabled ? "IPSEC" : null
-  type                     = "DEDICATED"

```

Comment:
Will `type` auto-magically default to `PARTNER` when `edge_availability_domain` is provided?